src/quick-lint-js/parse.h

```diff
@@ -438,10 +452,21 @@ class parser {
         this->skip();
         break;
       default:
```

Comment:
Why not check labelled break statements and labelled continue statements too? Why only check non-labelled breaks and continues?